doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+## $PlatformShort$ Chart and Graph Types
+
+The $PlatformShort$ product has over 65 different chart and graph types for any scenario – from a single chart display to an interactive dashboard. You can create $PlatformShort$ charts like Pie, Bar, Area, Line, Point, Stacked, Doughnut, Scatter, Gauge, Polar, Treemap, Stock, Financial, Geospatial Maps and more for your mobile or web apps. The benefit of our $PlatformShort$ chart vs. others is full support for features like: 
+
+- Responsive Web Design built in 
+- Interactive Panning and Zooming with Mouse, Keyboard and Touch 
+- Full Control of Chart Animation 
+- Chart Drill-Down Events 
+- Real-Time Streaming Support 
+- High-Volume (Millions of Data Points) Support 
+- Trends Lines and other Data Analysis features 
+
+Built with a modular design of axis, markers, series, legend, and annotation layers, the $PlatformShort$ chart makes it easy to design a render any type of data story. Build a simple chart with a single data series, or build more complex data stories with multiple series of data, with multiple axis in composite views. 
+
+### $PlatformShort$ Bar Chart
+
+The $PlatformShort$ Bar Chart, or Bar Graph is among the most common category chart types used to quickly compare frequency, count, total, or average of data in different categories with data encoded by horizontal bars of equal width and differing lengths. They are ideal for showing variations in the value of an item over time, data distribution, sorted data ranking (high to low, worst to best). Data is represented using a collection of rectangles that extend from the left to right of the chart towards the values of data points.
+
+Learn more about our [bar charts](chart-types-bar.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### $PlatformShort$ Pie Chart
+
+The $PlatformShort$ Pie Chart, or Pie Graph, is a very common part-to-whole chart type. Part-to-whole charts show how categories (parts) of a data set add up to a total (whole) value. Categories are shown in proportion to other categories based on their value percentage to the total value being analyzed. A pie chart renders data values as sections in a circular, or pie-shaped graph. Each section, or pie slice, has an arc length proportional to its underlying data value. The total values represented by the pie slices represent a whole value, like 100 or 100%. Pie charts are perfect for small data sets and are easy to read at a quick glance.
+
+Learn more about our [pie charts](../pie-chart.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
```

Comment:
https://share.goabstract.com/02743744-824f-4699-954e-8f001e608fc0?collectionLayerId=72edf789-0ade-4dff-bdd2-634f5f75746c&mode=build&selected=3791049301-98D4E82D-3458-441C-9B36-7050E75E1875&sha=ca960725d69fa0932376c375959f9f1e81bcee9a

On click - if another part of the chart is clicked, the first one goes back in, and the second is exploded.